wcfsetup/install/files/lib/system/endpoint/controller/core/boxes/DisableBox.class.php

```diff
@@ -31,7 +30,9 @@ public function __invoke(ServerRequestInterface $request, array $variables): Res
 
         $this->assertBoxCanBeDisabled($box);
```

Comment:
```suggestion
        $this->assertBoxCanBeDisabled();
```
Remove the parameter from the `assertBoxCanBeDisabled` method!